src/_common/media-item/post/post.ts

```diff
@@ -49,6 +49,10 @@ export default class AppMediaItemPost extends Vue {
 	@Emit('fullscreen')
 	emitFullscreen(_mediaItem: MediaItem) {}
 
+	get shouldShowFullscreen() {
```

Comment:
We should almost always show fullscreen option, I think. We should just not show it if the media item height or width is less than 100px or something.

Reply:
Also, maybe call it "shouldShowFullscreenOption"?

Reply:
Changed it to check if the mediaItem height and width is >= 150, and the device height is set to be restricted (currently only used in feed view).